js/expiring_messages.js

```diff
@@ -11,8 +11,8 @@
         var expired = new Whisper.MessageCollection();
         expired.on('add', function(message) {
             console.log('message', message.get('sent_at'), 'expired');
-            message.destroy();
             message.getConversation().trigger('expired', message);
+            message.destroy();
```

Comment:
This is kind of a paranoid change, allowing the conversation to process the message expiration before we delete it from the database. Will add a comment.